src/microbe_stage/Spawners.cs

```diff
@@ -366,7 +364,11 @@ public CompoundCloudSpawner(Compound compound, CompoundCloudSystem clouds, float
         this.amount = amount;
     }
 
-    public override IEnumerable<ISpawned>? Spawn(Node worldNode, Vector3 location)
+    public override int BinomialN => (int)(amount / 50000);
+    public override float BinomialP => 0.1f;
```

Comment:
I'm definitely confused as to what these values represent in your algorithm, and why you give the these values.